src/_data/catalog/widgets.json

```diff
@@ -863,6 +995,17 @@
     "link": "https://api.flutter.dev/flutter/widgets/FittedBox-class.html",
     "image": "<img alt='' src='/assets/images/docs/catalog-widget-placeholder.png'>"
   },
+  {
+    "name": "Floating action button",
+    "description": "The FAB represents the most important action on a screen. It puts key actions within reach.",
```

Comment:
**issues:**
- Make consistent with other descriptions.
- Remove the circular definition.
```suggestion
    "description": "Clickable block that represents the most important action on a screen. It puts key actions within reach.",
```